### ISSUE TITLE
The vcs-diff tool needs correct dependency configuration

### ISSUE THREAD

**Issue by jkonecny12** [fedora-copr/logdetective#28]
As it can be seen here https://github.com/fedora-copr/logdetective/pull/23 there is a lot of issues raise from missing packages in the pylint environment running the tests. We need to configure vcs-diff tool to install missing dependencies, ideally run poetry before the pylint execution.

**Comment by jkonecny12** [comment]
Hi @praiskup assigning you to this task as you have the best knowledge about vcs solution.

**Comment by praiskup** [comment]
There's the `install_rpm_packages` option: https://github.com/fedora-copr/vcs-diff-lint-action/blob/861a4663fb6d6d02dd30c5586164098f20bc3d8a/README.md#install_rpm_packages

Anytime a new dependency is added, we can add a corresponding liter-time dependency.  Or we can simply `pylintrc` ignore the import errors, up to you.

**Comment by jpodivin** [comment]
> There's the `install_rpm_packages` option: https://github.com/fedora-copr/vcs-diff-lint-action/blob/861a4663fb6d6d02dd30c5586164098f20bc3d8a/README.md#install_rpm_packages
> 
> Anytime a new dependency is added, we can add a corresponding liter-time dependency. Or we can simply `pylintrc` ignore the import errors, up to you.

I don't believe all of our dependencies are in repositories, even if they were, this project is published on pipy, so rpms aren't really that relevant at this point. I'd rather have this disabled.

**Comment by jpodivin** [comment]
Linter was disabled and supplanted by alternatives.
